scripts/normalize-yaml

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift(File.expand_path('../lib', File.dirname(__FILE__)))
+require 'yaml_normalizer'
+require 'yaml'
+
+YamlNormalizer.run(ARGV)
```

Comment:
Do you think that the note in the `i18n_spec.rb` spec description for failures that points to the script is close enough? I don't want the README to get too noisy